src/ui/components/NetworkMonitor/index.tsx

```diff
@@ -1,5 +1,5 @@
-import SplitBox from "devtools/client/shared/components/splitter/SplitBox";
-import React, { useState } from "react";
+import SplitBox from "devtools-splitter";
```

Comment:
yeesh... we should really do something about two splitter components and this vendors file...

Reply:
For record-keeping: we did https://github.com/RecordReplay/devtools/pull/4906